android/src/main/java/com/swmansion/reanimated/nodes/Node.java

```diff
@@ -100,6 +100,9 @@ protected final void forceUpdateMemoizedValue(Object value) {
     markUpdated();
   }
 
+  public void onDrop() {
```

Comment:
I'm not happy with this approach. It's rather pointless to create a method and override it in exactly one place. If I were you, I'd rather use `instanceof` and `isKindOfClass`.
I'm rather a fan of changing the smallest possible number of files / classes.